containers/airflow/Dockerfile

```diff
@@ -16,5 +16,6 @@ RUN sed -i "s|PIP_EXTRA_INDEX_URL|${PIP_EXTRA_INDEX_URL}|" ${AIRFLOW_USER_HOME}/
  && rm /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/t*.py \
  && rm /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/docker*.py \
  && rm /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example*.py \
+ && pip install --user --upgrade pip setuptools wheel \
```

Comment:
Maybe leave a comment to explain this upgrade. This is not a typical line, so one day we can forget why we added it.